odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1257,9 +1260,12 @@ def distribute_branding(self, e, branding=None, parent_xpath='',
         node_path = e.get('data-oe-xpath')
         if node_path is None:
             node_path = "%s/%s[%d]" % (parent_xpath, e.tag, index_map[e.tag])
-        if branding and not (e.get('data-oe-model') or e.get('t-field')):
-            e.attrib.update(branding)
-            e.set('data-oe-xpath', node_path)
+        if branding:
+            if not (e.get('data-oe-model') or e.get('t-field')):
+                e.attrib.update(branding)
+                e.set('data-oe-xpath', node_path)
+            elif e.get('t-field'):
+                e.set('data-oe-field-xpath', node_path)
```

Comment:
```suggestion
            if e.get('t-field'):
                e.set('data-oe-field-xpath', node_path)
            elif not e.get('data-oe-model'):
                e.attrib.update(branding)
                e.set('data-oe-xpath', node_path)
            
```

Reply:
Indeed, got confused rewriting it 21023232 times :smile: